client/src/lib/attempt-result.js

```diff
@@ -256,18 +256,71 @@ export function autocompleteTimeAttemptResult(time) {
   return roundOver10Mins(time);
 }
 
+/**
+ * Checks if a given set of results is a new world record average and returns a corresponding boolean.
+ */
+export function isWorldRecordAverage(
+  attemptResults,
+  eventId,
+  officialWorldRecords
+) {
+  const newAverage = average(attemptResults, eventId);
```

Comment:
We calculate the average already outside this function, so instead i would do `isWorldRecord(attemptResult, eventId, type, officialWorldRecords)`. Where `attemptResult` is either single or average and it's given by `type`.

Reply:
I think I understand what you are saying. Please check the latest commit and see if that was the implementation and interface you were thinking of.